### DISCUSSION TITLE
Welcome to gamesense.is_CRACKED Discussions!

### DISCUSSION THREAD

**Discussion by h4xrOx** [h4xrOx/GameSense.is.crack.CR4cKCRACK#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bajnn** [comment]
Hi there I'm a java developer that is currently learning ethical hacking.

How long did it take to crack/hack this?

Why do you hack/crack stuff is it for fun?

Why do you target cheats?

**Reply by h4xrOx** [reply]
The gamesense.is said it cracked it from the other .pub . The .is is
owned by the neverlose.cc, the neverlose.cc is many cheats, the same base, many sites and
rebranded as marketing scheme targeting children. The hosts, are many
hosts, who mostly do not care about the cheats they stole, not cracked,
from those who choose them for hosting. The same sites that sell the
leaked credentials once these things are dumped or cracked; are the same
hosts. I do it for free because my ideology is not aligned with bad
actor's. I do not believe information should be sold. Economy is only
enslaving progress by segregating information and our brightest minds. Those children scammed enjoy the cheats yes. I do not even play the games; albeit, my agenda is met when my ideology spreads to the youth. Charity is a beautiful thing for open source when
all external motivation is removed; a group of people working together peak in creativity. I
wish the world would seek what they will to create; rather than slave
for profit helping achieve agendas that has impeded progress for thousands of years. For the whole world is set back by a few people's greed. h4xr0x
isn't a marketplace, it's information I covet; not monetization! I do it
for you, and anyone else who is reading it.  I encourage all to do it
better than me in the spirit of the same idea.

**Comment by mgrs234** [comment]
how long did it take you to crack this?

**Reply by mgrs234** [reply]
did anyone help you to crack it?

**Reply by h4xrOx** [reply]
I live in Wisconsin. My Milwaukee Bucks are Champions so I'm going to answer this in  "Antetokounmpo".  I took advantage of the dev's "Ego"  that is their past ("the best reverse engineers in the world"). The best engineers in the world do not have a secure plan for walking 0day's.( that is the human element) I invoked their  "Pride" that is their future, ("we dominate HvH"). People who claim as much leave themselves vulnerable to "Humility", that is their Present.  A Social Engineering campaign that was successful partly due to  Ego presently blinding the administrations craving  for affirmation to sustain their Pride. (Future reputation of "the best reverse engineers in the world") Pride always comes before the fall. That said;  while they were laughing at my "crack " , " reversing it and proclaiming to their "HvH" community that h4xr0x doesn't know what he's talking about; I was kicking their back door in with humility. (thank you OAuth, Firebase, Azure, NodeJS and MS graph integration).  While  I do this for hobby, due to ideology; and not for monetary gain or fame - I would have taken my time in coming out with this. However to answer your last question, "did anyone help you to crack it ?", another target who's Ego was overcome by humility, was spending upwards 4 weeks reversing this. I actually give credit to Esotk/gamesense.pub for his contributions and commits in revenge. Without watching and stealing his work I wouldn't have had this content and conclusions as fast as I would have. That said; I  own private commits, source, and intimate knowledge of GameSense.pub, Neverlose.cc/GameSense.is the Hosts, creators and affiliates. .... Which I will use to give free content to members of my website; and maybe one day I will make all public; like the Aimware Crack my members requested that started this CSGO adventure for me. ( I do not play csgo, never have).                                                                            **As for Antetokounmpo** https://www.inc.com/jeff-haden/giannis-antetokounmpo-nba-championship-winning-mindset-stay-humble-in-moment.html  #FEAR_THE_DEAR

**Comment by Luckling** [comment]
When I opened the syringe shows both Please re - download the launcher

**Reply by Luckling** [reply]
How to deal it
